server/src/main/java/io/druid/segment/realtime/firehose/PredicateFirehose.java

```diff
@@ -31,6 +32,10 @@
  */
 public class PredicateFirehose implements Firehose
 {
+  private static final Logger log = new Logger(PredicateFirehose.class);
+  private static final int IGNORE_THRESHOLD = 5000;
+  private static long ignored = 0;
```

Comment:
good catch this should not be static but why does it matter if it is long and it is only counting the ignored rows